l3kernel/l3str-convert.dtx

```diff
@@ -317,6 +319,17 @@
 \prop_gput:Nnn \g_@@_alias_prop { utf32le } { utf32 }
 \prop_gput:Nnn \g_@@_alias_prop { utf32be } { utf32 }
 \prop_gput:Nnn \g_@@_alias_prop { hexadecimal } { hex }
+\bool_lazy_any:nTF
+  {
+    \sys_if_engine_luatex_p:
+    \sys_if_engine_xetex_p:
+  }
+  {
+    \prop_gput:Nnn \g__str_alias_prop { dflt } {  }
+  }
+  {
+    \prop_gput:Nnn \g__str_alias_prop { dflt } { utf8 }
```

Comment:
`@@` instead of `_str`

Reply:
grr, that happens if you copy from a test file. Corrected